test/unit/grammar/range_rule.cpp

```diff
@@ -75,7 +75,7 @@ struct range_rule_test
     void
     testRange()
     {
-        constexpr auto r0 = range_rule(
```

Comment:
Do we have a plan for `constexpr` rules that are inline in headers that depend on ranges? Or we don't need a plan?